files/es/web/javascript/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: JavaScript
+slug: Web/JavaScript
+tags:
+ - Aprender
+ - JavaScript
+ - 'l10n:priority'
+translation_of: Web/JavaScript
+---
+{{JsSidebar}}
+
+**JavaScript** (**JS**) es un lenguaje de programación ligero, interpretado, o compilado [justo-a-tiempo](https://en.wikipedia.org/wiki/Just-in-time_compilation) (just-in-time) con {{Glossary("First-class Function", "funciones de primera clase")}}. Si bien es más conocido como un lenguaje de scripting (secuencias de comandos) para páginas web, y es usado en [muchos entornos fuera del navegador](https://en.wikipedia.org/wiki/JavaScript#Uses_outside_Web_pages), tal como {{Glossary("Node.js")}}, [Apache CouchDB](https://couchdb.apache.org/) y [Adobe Acrobat](https://www.adobe.com/devnet/acrobat/javascript.html) JavaScript es un lenguaje de {{Glossary ("Prototype-based programming", "programación basada en prototipos")}}, multiparadigma, de un solo hilo, dinámico, con soporte para programación orientada a objetos, imperativa y declarativa (por ejemplo programación funcional). Lee más en [acerca de JavaScript](/es/docs/Web/JavaScript/About_JavaScript).
```

Comment:
Some external URLs/resources in English also have translated versions. If you find that they're good quality, feel free to link to the Spanish version of that resource rather than the English one :)

I only checked a couple of the Wikipedia pages, but these look like they *could* be linked to in place of the English ones (this is, of course, completely up to you):

- https://es.wikipedia.org/wiki/Java_(lenguaje_de_programaci%C3%B3n)
- https://es.wikipedia.org/wiki/JavaScript

Reply:
it totally makes sense, I also agree, I'll share this "small but minute detail" with the rest of the community. 

I'll add the suggestion for you, why don't have permission to suggest edits?

Reply:
Oops, that was a mix-up on my part. I forgot the syntax for adding suggestions and for some reason thought that doing so required specific permissions, I'll edit them now :)

Reply:
hey @cw118 taking advantage you are here, we were wondering if we could add an ISSUE_TEMPLATE for the Spanish version, it could be challenging for Spanisk Speakers to open a new issue and see the questions in English, we are thinking in maybe add the same questionary but translated to Spanish, what do you think?

Reply:
I might not be the best person to ask (maybe a question for other maintainers/review teams?), but I personally think that'd be great :) 

And just to avoid filling up this PR/thread with lots of comments, would it be alright if I join Spanish MDN matrix chat? It'd be the perfect platform for discussions like these (and now I really have to learn Spanish)

Reply:
Hi @cw118. I am new at this and I appreciate your help. Thank you for helping us to have a more presentable PR (I hope I have expressed myself well) , I can't be more agree about your proposal to use links in Spanish instead of the English one.

Reply:
**Just some important stuff to add to the external links detail**: 

- Some resources, including official specifications, are only available in English (e.g. `https://tc39.es/ecma402/`), so localizers just use the English resource
- Even with others where translations *are* available, they might not be "good enough quality" to include — this is up to the community (contributors and reviewers) to verify — in which case it could be better to stick with the original English link. 

I hope I explained that okay. Either way, if it's alright with everyone, I think I'm going to join the `l10n-es` Matrix chat and we could discuss everything further there?

> I am new at this and I appreciate your help. Thank you for helping us to have a more presentable PR (I hope I have expressed myself well)

Not a problem, it's so nice to see the Spanish community back in action here! You guys are doing great :)